packages/flutter_tools/lib/src/commands/precache.dart

```diff
@@ -9,6 +9,7 @@ import '../globals.dart';
 import '../runner/flutter_command.dart';
 import '../version.dart';
 
+// TODO!
```

Comment:
Should be handled automatically via the `DevelopmentArtifact.iOS`

Reply:
Cool